### ISSUE TITLE
Issue with FBref:  500 Server Error: Internal Server Error for url

### ISSUE THREAD

**Issue by yousseflamrini** [probberechts/soccerdata#710]
Hello, firt i would like to thank you about this usefull library.
Recently i was trying to scrape the team match stats of FBref, the script runs for multiple minutes, and suddenly i got this error (check attached screenshot), apparently the requested url is not present anymore. you can find attached also the used script.
![Bug_Error](https://github.com/user-attachments/assets/1507439b-33bb-43db-bab9-47bf789b18f6)
![Used_Script](https://github.com/user-attachments/assets/e3f544ab-b0b5-4b07-b08c-b5f8cbab9e5c)

**Comment by lorenzodb1** [comment]
Have you tried clearing the cache?

**Comment by yousseflamrini** [comment]
Yes, but i think this error is linked to the requested url, when i try to open it manually, the FBref website says that this page is not available : 

Here is the web browser result when typing this url (https://fbref.com/en/squads/198ca51f/1923/matchlogs/all_comps/schedule):
![Webbrowser_result](https://github.com/user-attachments/assets/e6dd4e68-f09d-4505-8586-28b03698c3f2)

**Comment by ilyacherevkov** [comment]
Is there an option to skip the error and proceed to the next entry? As of now, the whole process is crashing after a few unsuccessful attempts.

**Comment by probberechts** [comment]
Closing this, since FBref has completely changed since and this is probably no longer relevant.
